CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ Each issue fixed should contain one bullet summarizing the work done.
 <hr />
 
 ## [Unreleased]
+* [CIO-257](https://github.com/jwu910/check-it-out/issues/257) 
+Adding Table and Statusbar container only on promise resolution which is returned from fetching git refs.
+Moved all the screen appends to happen on the initial load.
+Removed ``loadingDoalog`` from line ``118`` as we are already ``calling loadingDialog.stop()`` after successful checkout.
```

Comment:
This could be simplified to just "Fix append order to address missing load screen bug"

Changelogs can usually be kept clean and on one line as they should just be a summary of the work done. :+1: